modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBArticleItemSelectorViewDisplayContext.java

```diff
@@ -219,63 +220,58 @@ public KBArticle getLatestArticle(KBArticle kbArticle) {
 	}
 
 	public List<BreadcrumbEntry> getPortletBreadcrumbEntries()
-		throws Exception {
+		throws PortalException, PortletException {
```

Comment:
I myself prefer to explicitly declare all unchecked exceptions as you're doing here, but I think Brian will ask you change it back to Exception.